README.adoc

```diff
@@ -43,6 +43,21 @@ This mode of operation will be used in production.
 
 To build and run the non-simulated version of Omicron, see: xref:docs/how-to-run.adoc[].
 
+=== Run tests with nextest
+
+The supported way to run tests is via https://nexte.st/[cargo-nextest].
+
+If you don't already have nextest installed, get started by https://nexte.st/book/pre-built-binaries[downloading a pre-built binary] or installing nextest via your package manager. Nextest has pre-built binaries for Linux, macOS and illumos.
+
+Then, run tests with:
+
+[source,text]
+----
+$ cargo nextest run
+----
```

Comment:
It would be redundant with the nextest docs, but I know my first question here would be how do I run the tests for one crate, `cargo nextest run -p omicron-nexus`.